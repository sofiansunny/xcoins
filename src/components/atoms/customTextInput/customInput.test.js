import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import CustomTextInput from '.';
describe('SomeComponent component', () => {
  it('Shallow rendering', () => {
    const props = {
      style: {},
      containerStyle: {},
      LeftComponent: null,
      RightComponent: null,
      placeholderText: null,
    };
    const wrapper = shallow(<CustomTextInput {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
