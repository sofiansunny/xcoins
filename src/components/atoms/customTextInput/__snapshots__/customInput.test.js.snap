// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomeComponent component Shallow rendering 1`] = `
<View
  style={
    Array [
      Object {
        "borderColor": "#ffffff",
        "flexDirection": "row",
      },
      Object {},
    ]
  }
>
  <TextInput
    onChangeText={[Function]}
    placeholder={null}
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "color": "#000000",
          "fontSize": 31.428571428571427,
          "padding": 0,
        },
        Object {
          "flex": 1,
        },
        Object {},
      ]
    }
    value="undefined"
  />
</View>
`;
